Examples/Tests/restart/inputs_test_2d_runtime_components_picmi.py

```diff
@@ -151,3 +171,6 @@ def add_particles():
 ##########################
 
 sim.step(1)
+
+if not sim.amr_restart:
+    del electrons
```

Comment:
Why not always delete it?
```suggestion
del electrons
```

Reply:
If I just have `del electrons` all the tests pass indeed, but pre-commit complains `Undefined name electrons`.

Reply:
Oh, interesting! Then leave as is for now of course.